crypto/src/test_utils/test_conversation/commit.rs

```diff
@@ -0,0 +1,290 @@
+    pub async fn e2ei_rotate(self, credential_bundle: Option<&CredentialBundle>) -> TestConversation<'a> {
+        self.e2ei_rotate_guarded(credential_bundle).await.notify_members().await
+    }
+
+    /// Create a rotate commit that hasn't been merged by the actor.
+    /// On [OperationGuard::notify_members], the actor will receive this commit.
+    pub async fn unmerged_e2ei_rotate(self, credential_bundle: &CredentialBundle) -> OperationGuard<'a, Commit> {
+        let mut conversation_guard = self.guard().await;
+        let conversation = conversation_guard.conversation().await;
+        let mut leaf_node = conversation.group.own_leaf().unwrap().clone();
+        drop(conversation);
+        leaf_node.set_credential_with_key(credential_bundle.to_mls_credential_with_key());
+        let commit = conversation_guard
+            .update_key_material_inner(Some(credential_bundle), Some(leaf_node))
+            .await
+            .unwrap()
+            .commit;
+
+        OperationGuard::new(TestOperation::Update, commit, self, [])
+    }
+
+    /// Like [Self::e2ei_rotate], but also when notifying other members, call [SessionContext::verify_sender_identity].
+    pub async fn e2ei_rotate_and_verify_sender(
+        self,
+        credential_bundle: Option<&CredentialBundle>,
+    ) -> TestConversation<'a> {
+        self.e2ei_rotate_guarded(credential_bundle)
+            .await
+            .notify_members_and_verify_sender()
+            .await
+    }
+
+    /// Guarded version of [Self::e2ei_rotate].
+    pub async fn e2ei_rotate_guarded(self, credential_bundle: Option<&CredentialBundle>) -> OperationGuard<'a, Commit> {
+        self.guard().await.e2ei_rotate(credential_bundle).await.unwrap();
+        let commit = self.transport().await.latest_commit_bundle().await.commit;
+        let committer_index = self.actor_index();
+        OperationGuard::new(TestOperation::Update, commit, self, [committer_index])
+    }
+
+    /// Commit all proposals pending in the actor's conversation state.
+    pub async fn commit_pending_proposals(self) -> TestConversation<'a> {
+        self.commit_pending_proposals_guarded().await.notify_members().await
+    }
+
+    /// Guarded version of [Self::commit_pending_proposals].
+    pub async fn commit_pending_proposals_guarded(self) -> OperationGuard<'a, Commit> {
+        self.guard().await.commit_pending_proposals().await.unwrap();
+        let commit = self.transport().await.latest_commit().await;
+        let actor_index = self.actor_index();
+        OperationGuard::new(
+            // Comitting pending proposals is equivalent to an update
+            TestOperation::Update,
+            commit,
+            self,
+            [actor_index],
+        )
+    }
+
+    /// Remove this member from this conversation.
+    ///
+    /// Applies the removal to all members of the conversation.
+    ///
+    /// Panics if you try to remove the current actor (by default, the conversation creator);
```

Comment:
Since we already refer to "members" in this function, I think we should also use "member" instead of "actor" here. Possibly also in other places.